src/jailer/src/chroot.rs

```diff
@@ -9,7 +9,7 @@ use std::ptr::null;
 use libc;
 
 use super::{to_cstring, Error, Result};
-use utils::syscall::SyscallReturnCode;
+use vmm_sys_util::syscall::SyscallReturnCode;
```

Comment:
Hi, thanks for your contribution! If you take a look at this discussion: https://github.com/firecracker-microvm/firecracker/pull/1996#discussion_r450015058, you can see that our `utils` crate works as a wrapper over `vmm_sys_util`, so for using `syscall` from `vmm_sys_util` instead of the local one, only this line: https://github.com/firecracker-microvm/firecracker/blob/master/src/utils/src/lib.rs#L8 should be modified.

Sorry for not adding details/explanations to the issue :(, it seemed @defunctio wants to do this change too and I thought it would happen quickly.

Reply:
Okay. Do you mean @defunctio wants to work on this issue ?

Reply:
The issues are not tightly coupled, you can work on it if you want, thanks! You will have to also update `vmm_sys_util` version with the latest one: 0.6.1 in `Cargo.toml` from utils.

Reply:
I am little bit confused as I am beginner in Rust. Can you please send me the invite to slack channel ?

Reply:
Sure, sorry for the late response, you can find the slack channel [here](https://github.com/firecracker-microvm/firecracker#faq--contact), you can ping me there and hopefully I will explain better :(.